pkg/server/serve.go

```diff
@@ -44,4 +44,8 @@ func Serve(ctx context.Context, hc *HTTPConfig, gc *GRPCConfig, s rekorServer) {
 	httpMetrics.start(&wg)
 
 	wg.Wait()
+
+	if err := tesseraShutdownFn(ctx); err != nil {
```

Comment:
Do we end up with good enough logging that we will be able to tell that we are now waiting for tessera to shutdown, or do we need to explicitly log that?